BlazarService/src/main/java/com/hubspot/blazar/integration/slack/SlackClient.java

```diff
@@ -68,9 +68,7 @@ public void sendMessage(final SlackMessage message) {
         throw Throwables.propagate(e);
       }
     }
-    if (message.getIconEmoji().isPresent()) {
-      requestBuilder.setFormParam("icon_emoji").to(message.getIconEmoji().get());
-    }
+    requestBuilder.setFormParam("icon_emoji").to(message.getIconEmoji().or(":fire:"));
```

Comment:
I run with slack in compact mode. Didn't realize some of the messages we send don't have icons.
Set it to the fire emoji instead of the grey robot it normally shows.